web/src/views/DandisetLandingView/ExternalDandisetServicesDialog.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <v-menu
+    offset-y
+    left
+  >
+    <template
+      #activator="{ on, attrs }"
+    >
+      <v-btn
+        id="external-dandiset-services"
+        outlined
+        block
+        v-bind="attrs"
+        v-on="on"
+      >
+        <v-icon
+          color="primary"
+          left
+        >
+          mdi-web
+        </v-icon>
+        <span>Open with</span>
+        <v-spacer />
+        <v-icon right>
+          mdi-chevron-down
+        </v-icon>
+      </v-btn>
+    </template>
+    <v-card
+    >
+      <v-card-title class="pb-0" style="min-width: fit-content;">
+        External services:
+      </v-card-title>
+      <v-list>
+        <v-tooltip
+          open-on-hover
+          left
+        >
+          <template #activator="{ on }">
+            <div
+            v-on="on"
+            >
+              <v-list-item
+                :href="openInNeurosift()"
+              >
+                <v-icon
+                  color="primary"
+                  left
+                  small
+                >
+                  mdi-web
+                </v-icon>
+                Neurosift
+              </v-list-item>
+            </div>
+          </template>
+          <span>Open the Dandiset in Neurosift</span>
+        </v-tooltip>
+      </v-list>
+    </v-card>
+  </v-menu>
+</template>
+<script setup lang="ts">
+import { computed } from 'vue';
+import { useDandisetStore } from '@/stores/dandiset';
+
+const store = useDandisetStore();
+
+const currentDandiset = computed(() => store.dandiset);
+
+const openInNeurosift = () => {
+  const dandisetUrl = currentDandiset.value?.metadata?.url;
+  if (!dandisetUrl) {
+    alert('Unexpected: No url field found in the metadata of the dandiset');
+    return;
+  }
+  const staging = dandisetUrl.startsWith('https://gui-staging.dandiarchive.org/');
```

Comment:
Is there a better way to decide on the "instance name" used to tell apart at the Vue client level not only from staging but also from LINC or temporary testing one?  attn @kabilar @aaronkanzer @jjnesbitt . If so we better use that one and overall enable this only for stock DANDI main and staging but not for anything else.

Reply:
I don't think that we have yet abstracted the instance name.  At present, it is hard-coded in the LINC fork.

Reply:
#1810 does abstract the DANDI main and staging deployments.